include/ipc-client.h

```diff
@@ -24,6 +24,10 @@ char *get_socketpath(void);
  */
 int ipc_open_socket(const char *socket_path);
 /**
+ * Issues a single IPC command without reading response.
+ */
+void ipc_single_command_no_response(int socketfd, uint32_t type, const char *payload, uint32_t *len);
```

Comment:
Is there an advantage to reading it then instead of here? The response is generally meant to be specific to that request.